wp-admin/css/colors/_admin.scss

```diff
@@ -273,26 +281,25 @@ ul#adminmenu > li.current > a.current:after {
 	color: $menu-icon;
 }
 
-#wpadminbar .ab-top-menu > li:hover > .ab-item,
-#wpadminbar .ab-top-menu > li.hover > .ab-item,
-#wpadminbar .ab-top-menu > li > .ab-item:focus,
+#wpadminbar:not(.mobile) .ab-top-menu > li:hover > .ab-item,
```

Comment:
Avoid using id selectors<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 5